ENG-2025 - Document the identity verification API and phone gating (#3764)
FusionAuth/fusionauth-site: astro/src/content/docs/lifecycle/manage-users/verification/_common-questions-gating.mdx

---
---
import Aside from 'src/components/Aside.astro';
import Breadcrumb from 'src/components/Breadcrumb.astro';

**Question**: Will all my existing users retroactively get emails or SMS messages for either gate once enabled?

> **Answer**:  No. The only edge case if existing users have unverified identities (meaing `verified: false` with `verifiedReason: "Pending"`), and gating is enabled, then those users will be required to verify those identities at their next login.

Reviewer: ```suggestion
**Answer**:  No. However, if existing users have unverified identities (meaing `verified: false` with `verifiedReason: "Pending"`), and gating is enabled, then those users will be required to verify those identities at their next login.
```

To double-check your users' email verification status, click on <Breadcrumb>Users -> Search Bar</Breadcrumb> and use this Elasticsearch query string: `verified: false`. Note, this will return if either `user` or `user.registrations[x]` is unverified. Users found by this query will be required to verify either their email or application registration once the gate is enabled. However, **they will not be emailed until they log in**. During the login process, they will be required to click on a link to have a verification email sent to them.

<Aside>
  As of version 1.99.9, it's not yet possible to search for unverified phone number identities.
</Aside>

**Question**: What if I have accidentally/inadvertently imported users with a `verified: false` value (either on the user or application registration)?  In other words, can I do this manually.

**Answer**: Currently, there is no way to change this in the Admin UI. However, there is an [open issue](https://github.com/FusionAuth/fusionauth-issues/issues/2897) adding this functionality. Please upvote if you believe this is needed.

> You can currently set `user.verified` within the [user API](/docs/apis/users).

Author: Don't think you can, so removed this. FYI @spwitt

> ---

Author: I'm unsure about keeping shared common questions here. They can't search for unverified phone users as of now.

Author: Added in 460270abec63a2aa6bc3de205b997ae299040ed8

import Breadcrumb from 'src/components/Breadcrumb.astro';

**Question**: Will all my existing users retroactively get emails or SMS messages for either gate once enabled?

> **Answer**:  No. More specifically, this is an accurate statement if you have **never** enabled email/phone verification and/or application registration verification and imported your users with `verified: true` and `verifiedReason: "Completed"` values (as applicable).

Author: @andrewpai - pushed 1a9c20f71c96a8191fb5ad95cdf83b00cfb458fe

> **Answer**:  No. More specifically, this is an accurate statement if you have **never** enabled email/phone verification and/or application registration verification and imported your users with `verified: true` with `verifiedReason: "Completed"` values (as applicable).

Author: Line 15 in this file caught me eye, might need to discuss this
